From 485deb8e9b83cf045a39c03268ed2b36048ca7c1 Mon Sep 17 00:00:00 2001
From: tgeisler <tgeisler@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 21 Nov 2013 15:50:18 +0000
Subject: [PATCH 0496/1077] MANTIS	0029910: DoorDevice: new value
 getter/setter "NEPLITE_MODBUS_VERSION" "NEPLITE_FIRMWARE_VERSION"
 "NEPLITE_FIRMWARE_REVISION"

git-svn-id: svn://localhost/SelfServiceCommon/trunk@616 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/doordev.idl | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/doordev.idl b/SelfServiceCommon/Massai/idl/Massai/doordev.idl
index de0dca8b..641af450 100644
--- a/SelfServiceCommon/Massai/idl/Massai/doordev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/doordev.idl
@@ -6,7 +6,7 @@
  * Contains module doordev, which is the platform-internal interface
  * definition to test the common use gate's door device component.
  *
- * @version 1.5
+ * @version 1.6
  *
  * @author Sven Bornemann
  *
@@ -148,6 +148,9 @@ module doordev
         CAP_BLOCKED_DETECT_TIMEOUT,     /**< A value "BLOCKED_DETECT_TIMEOUT" may be read/set */
         CAP_NEPLITE_HARDWARE_VERSION    /**< A value "NEPLITE_HARDWARE_VERSION" may be read/set */
         //CAP_PASS_COMPLETE_TIMEOUT     /**< A value "PASS_COMPLETE_TIMEOUT" may be read/set */
+        //CAP_NEPLITE_MODBUS_VERSION    /**< A value "NEPLITE_MODBUS_VERSION" may be read/set */
+        //CAP_NEPLITE_FIRMWARE_VERSION  /**< A value "NEPLITE_FIRMWARE_VERSION" may be read/set only on newer firmware <> 515 */
+        //CAP_NEPLITE_FIRMWARE_REVISION /**< A value "NEPLITE_FIRMWARE_REVISION" may be read/set only on newer firmware <> 515 */
         // To be continued...
     };
 
-- 
2.41.0.windows.1

